microsoft-edge/devtools-guide-chromium/whats-new/whats-new.md

```diff
@@ -0,0 +1,14 @@
+---
+title: What's new in Microsoft Edge DevTools
+description: New features and fixes in Microsoft Edge DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/14/2022
+---
+# What's new in Microsoft Edge DevTools
+
+Check these articles for the latest What's New announcements, features, fixes, and updates about experimental features in Microsoft Edge DevTools.
+
+* [Experimental features](../experimental-features/index.md)
```

Comment:
Do we need this bucket landing page? The article says "check these articles" but then the bullet list below only contains a single page to the experimental features page. I fear it might not be clear to everyone that "these articles" refer to the various Whats New pages in the TOC.
One way would be to add links to all Whats New pages in the bullet list. Another way would be to remove this page.
What do you think?

Reply:
I think codepo8 expressed agreement with me that this page is needed, so that if we want to point ppl to What's New without being forced to point to a specific, rapidly outdated particular version of What's New.   I consider that to be very important: we have to be able to point to a _bucket_, not be restricted to pointing to a specific topical feature article when we do not mean to point to a specific version/feature article.  This is the theme of this Information Architecture Glue PR.  So, deleting this page is a last resort or not an option.  I'd definitely rather maintain a list of , say, the last 5 What's New pages, then delete this parent-level, bucket-level, much needed type of destination page.

Reply:
Great point. I agree with this.